datasets/noaa-cdr/collections/sea-ice-concentration/template.json

```diff
@@ -0,0 +1,565 @@
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+      "raster:bands": [
+        {
+          "nodata": 0,
+          "data_type": "uint8",
+          "spatial_resolution": 25000.0
+        }
+      ],
+      "classification:bitfields": [
+        {
+          "name": "BT_weather_filter_applied",
+          "offset": 0,
+          "length": 1,
+          "classes": [
+            {
+              "name": "BT_weather_filter_not_applied",
+              "value": 0
+            },
+            {
+              "name": "BT_weather_filter_applied",
+              "value": 1
+            }
+          ]
+        },
+        {
+          "name": "NT_weather_filter_applied",
+          "offset": 1,
+          "length": 1,
+          "classes": [
+            {
+              "name": "NT_weather_filter_not_applied",
+              "value": 0
+            },
+            {
+              "name": "NT_weather_filter_applied",
+              "value": 1
+            }
+          ]
+        },
+        {
+          "name": "BT_land_spillover_filter_applied",
+          "offset": 2,
+          "length": 1,
+          "classes": [
+            {
+              "name": "BT_land_spillover_filter_not_applied",
+              "value": 0
+            },
+            {
+              "name": "BT_land_spillover_filter_applied",
+              "value": 1
+            }
+          ]
+        },
+        {
+          "name": "NT_land_spillover_filter_applied",
+          "offset": 3,
+          "length": 1,
+          "classes": [
+            {
+              "name": "NT_land_spillover_filter_not_applied",
+              "value": 0
+            },
+            {
+              "name": "NT_land_spillover_filter_applied",
+              "value": 1
+            }
+          ]
+        },
+        {
+          "name": "valid_ice_mask_applied",
+          "offset": 4,
+          "length": 1,
+          "classes": [
+            {
+              "name": "valid_ice_mask_not_applied",
+              "value": 0
+            },
+            {
+              "name": "valid_ice_mask_applied",
+              "value": 1
+            }
+          ]
+        },
+        {
+          "name": "spatial_interpolation_applied",
+          "offset": 5,
+          "length": 1,
+          "classes": [
+            {
+              "name": "spatial_interpolation_not_applied",
+              "value": 0
+            },
+            {
+              "name": "spatial_interpolation_applied",
+              "value": 1
+            }
+          ]
+        },
+        {
+          "name": "temporal_interpolation_applied",
+          "offset": 6,
+          "length": 1,
+          "classes": [
+            {
+              "name": "temporal_interpolation_not_applied",
+              "value": 0
+            },
+            {
+              "name": "temporal_interpolation_applied",
+              "value": 1
+            }
+          ]
+        },
+        {
+          "name": "",
```

Comment:
Just confirming that `name=""` here in in `classification:bitfields` is deliberate.